devicon.json

```diff
@@ -4248,6 +4248,31 @@
         "color": "#1FA6CA",
         "aliases": []
     },
+    {
+        "name": "materializecss",
+        "altnames": [
+            "materialize",
+            "materialize-css",
+            "materialize css"
+        ],
+        "tags": [
+            "framework",
+            "css",
+            "design",
+            "material-design",
+            "framework"
+        ],
+        "versions": {
+            "svg": [
+                "original"
```

Comment:
We should also add `plain` here.
```suggestion
                "original",
                "plain"
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable